packages/client/src/I18n.re

```diff
@@ -111,58 +109,58 @@ module Translations = {
 
-    | (English, Auth_Login_Submit) => "Login"
-    | (Swedish, Auth_Login_Submit) => {js|Logga in|js}
+    | (`ENGLISH, Auth_Login_Submit) => "Login"
+    | (`SWEDISH, Auth_Login_Submit) => {js|Logga in|js}
 
-    | (English, BookTrip_Button) => "Book trip"
-    | (Swedish, BookTrip_Button) => {js|Boka resa|js}
+    | (`ENGLISH, BookTrip_Button) => "Book trip"
+    | (`SWEDISH, BookTrip_Button) => {js|Boka resa|js}
 
-    | (English, BookTrip_Choose_DropdownLabel) => "Choose option"
-    | (Swedish, BookTrip_Choose_DropdownLabel) => {js|Välj alternativ|js}
+    | (`ENGLISH, BookTrip_Choose_DropdownLabel) => "Choose option"
+    | (`SWEDISH, BookTrip_Choose_DropdownLabel) => {js|Välj alternativ|js}
 
-    | (English, BookTrip_From_DropdownLabel) => "From"
-    | (Swedish, BookTrip_From_DropdownLabel) => {js|Från|js}
+    | (`ENGLISH, BookTrip_From_DropdownLabel) => "From"
+    | (`SWEDISH, BookTrip_From_DropdownLabel) => {js|Från|js}
 
-    | (English, BookTrip_To_DropdownLabel) => "To"
-    | (Swedish, BookTrip_To_DropdownLabel) => {js|Till|js}
+    | (`ENGLISH, BookTrip_To_DropdownLabel) => "To"
+    | (`SWEDISH, BookTrip_To_DropdownLabel) => {js|Till|js}
 
-    | (English, BookTrip_PrepareTrip_Button) => "Prepare trip"
-    | (Swedish, BookTrip_PrepareTrip_Button) => {js|Förbered bokning|js}
+    | (`ENGLISH, BookTrip_PrepareTrip_Button) => "Prepare trip"
+    | (`SWEDISH, BookTrip_PrepareTrip_Button) => {js|Förbered bokning|js}
 
-    | (English, BookTrip_TripPrepared_Message) => "This trip has been prepared. You will get a notification when the drone is ready to be loaded"
-    | (Swedish, BookTrip_TripPrepared_Message) => {js|Du har nu förberett din bokning. Vi notifierar dig när det är dags att packa drönaren.|js}
+    | (`ENGLISH, BookTrip_TripPrepared_Message) => "This trip has been prepared. You will get a notification when the drone is ready to be loaded"
+    | (`SWEDISH, BookTrip_TripPrepared_Message) => {js|Du har nu förberett din bokning. Vi notifierar dig när det är dags att packa drönaren.|js}
 
-    | (English, BookTrip_GoToOverview_Button) => {js|Go to overview|js}
-    | (Swedish, BookTrip_GoToOverview_Button) => {js|Gå till överblick|js}
+    | (`ENGLISH, BookTrip_GoToOverview_Button) => {js|Go to overview|js}
+    | (`SWEDISH, BookTrip_GoToOverview_Button) => {js|Gå till överblick|js}
 
-    | (English, DroneStatus_Loading_Position) => "Fetching the drones position"
-    | (Swedish, DroneStatus_Loading_Position) => {js|Laddar drönares position.|js}
+    | (`ENGLISH, DroneStatus_Loading_Position) => "Fetching the drones position"
+    | (`SWEDISH, DroneStatus_Loading_Position) => {js|Laddar drönares position.|js}
 
-    | (English, Language_Choose_DropdownLabel) => "Select language"
-    | (Swedish, Language_Choose_DropdownLabel) => {js|Välj språk|js}
+    | (`ENGLISH, Language_Choose_DropdownLabel) => "Select language"
+    | (`SWEDISH, Language_Choose_DropdownLabel) => {js|Välj språk|js}
```

Comment:
This is cool! I did not know that `enums` were automatically translated to polymorphic variants, nice! 💯